github-pr-reviewer/code_reviewer/github_code.py

```diff
@@ -0,0 +1,28 @@
+from github import Github
+
+from code_reviewer.reviewer import CodeReviewer
+
+class GithubAPI:
+    def __init__(self, access_token):
+        self.g = Github(access_token)
+        self.code_reviewer = CodeReviewer()
+    
+    def write_comments_for_pr(self, repo_name, pr_number):
+        repo = self.g.get_repo(repo_name)
+        pr = repo.get_pull(pr_number)
+        commit = repo.get_commit(pr.head.sha)
+        files = pr.get_files()
+        for file in files:
+            if file.status == 'added' or file.status == 'modified':
```

Comment:
Consider using a more descriptive variable name instead of 'file' to make the code more readable, like 'pr_file' or 'git_file'.